device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json

```diff
@@ -0,0 +1,334 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "phy1",
+      "address": "1",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy1_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
```

Comment:
is this really used? if not, we should have remove it.

Reply:
Bus_id is not really used. The value put here is to pass the parser checking: https://github.com/Azure/sonic-swss/blob/ec104c1ba079901a4af1fb72981d34e0069ceeac/gearsyncd/gearboxparser.cpp#L140